src/app/src/components/Layers/ParcelLayer.js

```diff
@@ -0,0 +1,69 @@
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { DynamicMapLayer } from 'esri-leaflet';
+
+import { useMapLayer } from '../../hooks';
+import { PARCELS_LAYER_URL, DATA_LAYERS } from '../../constants';
+
+const PARCELS_LAYER_STYLE = [
+    {
+        id: 0,
+        source: {
+            type: 'mapLayer',
+            mapLayerId: 0,
+        },
+        drawingInfo: {
+            renderer: {
+                type: 'simple',
+                symbol: {
+                    type: 'esriSMS',
+                    style: 'esriSMSDiamond',
+                    color: [183, 121, 31, 255], // var(--chakra-colors-yellow-600)
+                },
+            },
+        },
+    },
+    {
+        id: 1,
+        source: {
+            type: 'mapLayer',
+            mapLayerId: 1,
+        },
+        drawingInfo: {
+            renderer: {
+                type: 'simple',
+                symbol: {
+                    type: 'esriSFS',
+                    style: 'esriSFSSolid',
+                    color: [0, 0, 0, 0], // No fill
+                    outline: {
+                        type: 'esriSLS',
+                        style: 'esriSLSSolid',
+                        color: [183, 121, 31, 255], // var(--chakra-colors-yellow-600)
+                        width: 1.0,
+                    },
+                },
+            },
+        },
+    },
+];
+
+export default function ParcelLayer() {
+    const layers = useSelector(state => state.map.layers);
+    const showLayer = useMemo(() => layers.includes('PARCELS'), [layers]);
+
+    return showLayer ? <RenderEsriLayer /> : null;
+}
+
+function RenderEsriLayer() {
+    useMapLayer(
+        new DynamicMapLayer({
+            url: PARCELS_LAYER_URL,
+            f: 'image',
+            layers: [0, 1],
```

Comment:
Layer 0 is the Parcel Centroids, Layer 1 is the Parcel Boundaries

Reply:
Curios for @gatesgodin input, but I feel like the parcel centroids layer won't be useful for this application.

Reply:
@caseycesari @rajadain I agree, parcel centroids are not very useful in this use case. Showing the parcel boundaries at the city or town level should be adequate. Colours and styling looks good at the parcel level.